self-reflection-1-skygennette.qmd

```diff
@@ -0,0 +1,63 @@
+---
+title: "Self Reflection 1"
+author: "Sky Gennette"
+format: html
+css: style_textcolor.css
+---
+
+## About you
+
+1.  Please tell me a little bit about your research interests and how you imagine integrating spatial data in your research.
+
+I'm interested in using remote sensing and artificial intelligence as tools to improve ecological monitoring. I plant on integrating spatially explicit spectral data derived from remote sensing platforms including satellites and Unoccupied Aerial Systems. 
+
+2.  How do you learn best, and what do you feel is your biggest strength as a student?
+
+Especially in the context of this type of course, I learn best by walking through code and annotating during class. I feel that my biggest strength is my ability to work and problem-solve independently.
+
+
+3.  If you want to, you can set a goal for this semester for something you want to work on in the context of this course (outside of the topic matter itself) -- for example, using StackOverflow to get answers to analysis questions or using git to track progress on homeworks. If you choose to set this type of goal, what can you do to help support yourself in working toward that goal? How can I help you to meet this goal?
+
+I want to try to make my code from this and other courses easier for myself to reference back to in the future when trying to find specific pieces of code. This might be creating separate cheatsheet documents or just a good repository in github with readme files. If you know of good systems for this, that would be helpful.
```

Comment:
I can think of some options for this. 

One would be to keep a folder of reference files based on classroom coding exercises and homework. You could copy-paste the different functions you use, add some explanation, and create separate files in a system that would work for you. 

If you wanted to make this more formal, part of your final project could be creating a cheatsheet/reference sheet for the code you use for that project and that could be part of your graded goals. 

Finally, the class website will always be published for your reference. You can always reference the website, or I could publish any cheatsheets you write for everyone in the Lessons tab.